src/node_code_integrity.h

```diff
@@ -0,0 +1,83 @@
+// Windows API documentation for WLDP can be found at
+// https://learn.microsoft.com/en-us/windows/win32/api/wldp/
+
+#ifndef SRC_NODE_CODE_INTEGRITY_H_
+#define SRC_NODE_CODE_INTEGRITY_H_
```

Comment:
inside this define, we need the following, as same as other header files in the project:

#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS